_posts/2021-03-29-changelog-144.markdown

```diff
@@ -0,0 +1,81 @@
+---
+layout: post
+title: "IntelliJ Rust Changelog #144"
+date: 2021-03-29 17:00:00 +0300
+---
+
+## New Features
+
+<!-- https://github.com/intellij-rust/intellij-rust/pull/6956 -->
+* Increase minimal supported rustc version to 1.41.0
+
+<!-- https://github.com/intellij-rust/intellij-rust/pull/6907 -->
+* Provide debugging with LLDB in [Rider for Unreal Engine](https://www.jetbrains.com/lp/rider-unreal/) 2021.1
+
+<!-- https://github.com/intellij-rust/intellij-rust/pull/6799 -->
+* Complete dependencies by subwords in `Cargo.toml` when using experimental local crates index. This feature is disabled by default
+  for now. To use it, enable `org.rust.crates.local.index` experimental feature
```

Comment:
Let's add a note on how to enable experimental feature. It's quite frequent user question